R/utils.R

```diff
@@ -25,6 +25,14 @@ if (base::getRversion() < "3.2.0") {  # Apr 2015
   isNamespaceLoaded = function(x) x %chin% loadedNamespaces()
 }
 
+# R 3.3.0 [April 2016]
+if (!exists('startsWith', as.environment('package:base'))) {
```

Comment:
It unlikely matters, but it may be slightly more robust to pass `inherits=FALSE` to `exists()` here.
For example:  `exists("startsWith", "package:utils")` returns TRUE even though `startsWith` is in `base`, but `exists("startsWith", "package:utils", inherits=FALSE)` is FALSE as expected.
But do we care that `startsWith` is in `base` and not moved to `utils` in future? No. Maybe just `exists("startsWith")` then. I don't think that would pick up `startsWith` defined in another package the user has loaded because packages have a namespace which protects their internals from seeing functions defined in other packages. However I'm not quite sure about utils vs base; i.e. whether `exists("startsWith")` would find it if it is moved to `utils`.
Not something to worry about.  I'll just add the `inherits=FALSE` and be done with it.

Reply:
Good point. I think you're right that it doesn't matter but I agree it's probably better practice to use inherits=FALSE